domain/domain.go

```diff
@@ -575,6 +575,29 @@ func (do *Domain) UpdateTableStatsLoop(ctx context.Context) error {
 			}
 		}
 	}(do)
+	go func(do *Domain) {
+		id := do.ddl.OwnerManager().ID()
+		cancelCtx, cancelFunc := goctx.WithCancel(goctx.Background())
+		var statsOwner ddl.OwnerManager
+		if do.etcdClient == nil {
+			statsOwner = ddl.NewMockOwnerManager(id, cancelFunc)
+		} else {
+			statsOwner = ddl.NewOwnerManager(do.etcdClient, statistics.StatsPrompt, id, statistics.StatsOwnerKey, cancelFunc)
+		}
+		err := statsOwner.CampaignOwner(cancelCtx)
+		if err != nil {
```

Comment:
Then there will be no auto analyze since there is no owner. Or do we have better solutions?